PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,10 @@
 
 What features does this change enable? What bugs does this change fix? High-Level description of how things changed.
 
+## Related JIRA
+
+Please link the JIRA issue requiring this bump so it can be closed once the PR is merged.
```

Comment:
I thought that's what the "Related Issues" section below is for. What's the difference?

Reply:
The reason for this that we want to have a dedicated section to link the corresponding JIRA bump ticket. For example, the Related JIRA is a new Marathon version bump, and the Related tickets are any JIRA tickets which have been resolved with this bump.

Reply:
We probably should run with a different title to clearly state the difference between these two sections. How about `Associated Issues` of `Linked Issues`?

```markdown

## Associated Issues

<!-- Please link all issue associated (e.g., bump or release tracking issues) with this PR. --> 

- [DCOS_OSS-<number>](https://jira.dcos.io/browse/DCOS_OSS-<number>) Update Foo and Bar. 


## Related Issues

<!-- Please link all related issue (e.g., tasks and bugs) addressed with this PR. --> 

- [DCOS_OSS-<number>](https://jira.dcos.io/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.